src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -16,6 +16,10 @@
             "deleted": "Deleted Programming Exercise with title {{ param }}",
             "delete": {
                 "question": "Are you sure you want to delete the Programming Exercise <strong>{{ title }}</strong>?",
+                "questionLocalVC": {
+                    "single": "Are you sure you want to delete the Programming Exercise <strong>{{ title }}</strong>?<br>All student, template, solution, test, and auxiliary repositories and build plans will be deleted.<br>Be careful: this can <strong class='text-danger'>NOT</strong> be undone!<br>Before you activate this option, we recommend that you archive the exercise first!",
+                    "multiple": "Are you sure you want to delete selected programming exercises?<br>All student, template, solution, test, and auxiliary repositories and build plans will be deleted.<br>Be careful: this can <strong class='text-danger'>NOT</strong> be undone!<br>Before you activate this option, we recommend that you archive the exercise first!"
```

Comment:
"can not" -> "cannot"

Also, in English we have "...that you archive the exercise first!" - but in German, we have "...die Repositories und Build-Pläne erst zu sichern!" - can you double-check?

Reply:
you're right, it wasn't consistent. I've adjusted all translations to "archive the repositories and buildplans"